### ISSUE TITLE
Loader fails on import of large RegML file

### ISSUE THREAD

**Issue by chosak** [cfpb/eregs-2.0#37]
Running import on a large RegML file seems to cause an error during XML parsing.

Possible workaround: use [`huge_tree=True`](http://lxml.de/api/lxml.etree.XMLParser-class.html)?

## Current behavior

Running `import_xml` on a large RegML file fails.

## Expected behavior

Running `import_xml` on a large RegML file succeeds.

## Steps to replicate behavior (include URLs)

1. Run this command:

  ```sh
$ ./manage.py import_xml ../regulations-xml/regulation/1026/2013-13173.xml
...
  File "Projects/eregs-2.0/eregs_core/management/commands/import_xml.py", line 23, in handle
    xml_tree = etree.fromstring(xml)
  File "src/lxml/lxml.etree.pyx", line 3228, in lxml.etree.fromstring (src/lxml/lxml.etree.c:79594)
  File "src/lxml/parser.pxi", line 1848, in lxml.etree._parseMemoryDocument (src/lxml/lxml.etree.c:119113)
  File "src/lxml/parser.pxi", line 1736, in lxml.etree._parseDoc (src/lxml/lxml.etree.c:117793)
  File "src/lxml/parser.pxi", line 1102, in lxml.etree._BaseParser._parseDoc (src/lxml/lxml.etree.c:112037)
  File "src/lxml/parser.pxi", line 595, in lxml.etree._ParserContext._handleParseResultDoc (src/lxml/lxml.etree.c:105881)
  File "src/lxml/parser.pxi", line 706, in lxml.etree._handleParseResult (src/lxml/lxml.etree.c:107589)
  File "src/lxml/parser.pxi", line 635, in lxml.etree._raiseParseError (src/lxml/lxml.etree.c:106443)
  File "<string>", line 39312
lxml.etree.XMLSyntaxError: internal error: Huge input lookup, line 39312, column 2006
```

I've observed this on a number of Reg Z files.

**Comment by grapesmoker** [comment]
Yes, I think `huge_tree=True` is the way to go. This is how we do notice application and it works fine.

**Comment by chosak** [comment]
This was fixed in #54.
